brave/brave-core
# Refactored supported regions, large anonymity country codes and other country codes to use std::set

Resolves https://github.com/brave/brave-browser/issues/9205

## Submitter Checklist:

- Verified that these changes build without errors on
- Verified that these changes pass automated tests (unit, browser, security tests) on

## Test Plan:

- Confirm ads is enabled for supported regions
- Confirm ads is disabled for unsupported regions
- Confirm https://github.com/brave/brave-core/pull/5067 is working as expected

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on.
